app/components/support_form_component.html.erb

```diff
@@ -0,0 +1,75 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l"><%= school.name if school.present? %></span>
+    <h1 class="govuk-heading-l">Contact support</h1>
+
+    <p class="govuk-body">
+      <%= t("support_form.introduction.#{subject}", participant_name: participant_profile_full_name) %>
+    </p>
+
+    <p class="govuk-body">We’ll automatically include these details for you: </p>
+
+    <dl class="govuk-summary-list">
```

Comment:
Any reason not to use the [`summary_list` ruby component](https://govuk-components.netlify.app/components/summary-list/) here?

Reply:
Not really, it works either way.